tests/e2e/util/kubeUtils.go

```diff
@@ -67,6 +67,17 @@ func DeleteNamespace(n string) error {
 	return err
 }
 
+// NamespaceDeleted check if a kubernete namespace is deleted
+func NamespaceDeleted(n string) (bool, error) {
+	output, err := Shell("kubectl get namespace %s", n)
+	if strings.Contains(output, "NotFound") {
```

Comment:
I didn't put namespace as it is supposed to only return the namespace being requested to get. I didn't want to add detailed string checking as I didn't want to change the test code if the message happens to be changed from k8s.